app/models/topic.rb

```diff
@@ -0,0 +1,4 @@
+class Topic < ApplicationRecord
+	has_many :topictaggings
```

Comment:
You can fix in local, stage the change, and do a `git commit --amend` or a rebase, then `git push --force`. This will also let you change the commit message.